semver/test_comparator.ts

```diff
@@ -7,6 +7,7 @@ import { cmp } from "./cmp.ts";
  * @param version The version to compare
  * @param comparator The comparator
  * @returns True if the version is within the comparators set otherwise false
+ * @deprecated (will be removed in 0.212.0) Use {@linkcode compare} instead.
```

Comment:
Nit: 0.212.0 is quite soon. It would mean the lifetime of the deprecation would be one minor version. I think we should at least push out to 0.213.0.

Reply:
done